src/actions/guides/frontend/html_forms.cr

```diff
@@ -1,4 +1,5 @@
 class Guides::Frontend::HtmlForms < GuideAction
+  ANCHOR_SHARED_COMPONENTS = "shared-components"
```

Comment:
All of the others we've prefixed with `perma-`.. I don't remember why though.

Reply:
Probably me overthinking things :P I'll add perma to match the others